test/fixtures/.ignore-i18n

```diff
@@ -0,0 +1 @@
+**/hello.vue
```

Comment:
Could you test with the fixture file path? (for example, `src/pages/Login.vue`)
`**/hello.vue` is not the fixture file path.
In unit tests, We need to use the fixture file path.

Reply:
Thank you for your review!
Please check my commit!